lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -45,27 +46,110 @@
 from improver.tests.set_up_test_cubes import set_up_variable_cube
 from improver.utilities.warnings_handler import ManageWarnings
 
+try:
+    imp.find_module('statsmodels')
+    STATSMODELS_FOUND = True
+except ImportError:
+    STATSMODELS_FOUND = False
+
+
 IGNORED_MESSAGES = ["Collapsing a non-contiguous coordinate.",
                     "Not importing directory .*sphinxcontrib'",
                     "The pandas.core.datetools module is deprecated",
                     "numpy.dtype size changed",
                     "The statsmodels can not be imported",
                     "invalid escape sequence",
                     "can't resolve package from",
-                    "Collapsing a non-contiguous coordinate.",
                     "Minimisation did not result in"
                     " convergence",
                     "\nThe final iteration resulted in a percentage "
                     "change that is greater than the"
-                    " accepted threshold "]
+                    " accepted threshold ",
+                    "divide by zero encountered in true_divide",
```

Comment:
Where do these new warnings come from?

Reply:
The `divide by zero encountered in true_divide` is coming from the CRPS minimisation in the line: `if not np.isfinite(np.min(mu/sigma)):` implying that `sigma` is zero in this case. This is ok in this case, as the minimisation will just try again will slightly different coefficients, if a particular set of coefficients has resulted in a `sigma` of zero. 

The `invalid value encountered for` is occurs in multiple places, but I think that this is again related to the minimisation with runtime warnings of invalid value encountered in greater, invalid value encountered in less, invalid value encountered in true_divide, invalid value encountered in reduce, etc, which are arising from the minimisation.

Reply:
Ok, I'm happy for these warnings to be ingored